evap/staff/importers/base.py

```diff
@@ -13,6 +14,11 @@
 from django.utils.translation import gettext as _
 from django.utils.translation import gettext_lazy, ngettext
 
+from evap.evaluation.models import UserProfile
+
+if typing.TYPE_CHECKING:
+    from evap.staff.importers.user import UserData
```

Comment:
I'd prefer preventing circular imports by putting all the new stuff in `importers/decisions.py` or similar